packages/common/test/search/_internal/portalSearchGroupMembers.test.ts

```diff
@@ -0,0 +1,335 @@
+import { IHubSearchOptions, IQuery, cloneObject } from "../../../src";
+import { portalSearchGroupMembers } from "../../../src/search/_internal/portalSearchGroupMembers";
+import * as Portal from "@esri/arcgis-rest-portal";
+import { MOCK_AUTH } from "../../mocks/mock-auth";
+import * as users from "../../../src/users";
+
+describe("portalSearchGroupMembers:", () => {
+  describe("throws if not passed IQuery.properties.group:", () => {
+    let qry: IQuery;
+    beforeEach(() => {
+      qry = {
+        targetEntity: "groupMember",
+        filters: [
+          {
+            predicates: [
+              {
+                name: "steve",
+              },
+            ],
+          },
+        ],
+      };
+    });
+    it("no properties passed", async () => {
+      const opts: IHubSearchOptions = {};
+      try {
+        await portalSearchGroupMembers(qry, opts);
+      } catch (err) {
+        expect(err.message).toEqual(
+          "Group Id required. Please pass as query.properties.groupId"
+        );
+      }
+    });
+    it("passing query.properties", async () => {
+      const opts: IHubSearchOptions = {};
+      qry.properties = {};
+      try {
+        await portalSearchGroupMembers(qry, opts);
+      } catch (err) {
+        expect(err.message).toEqual(
+          "Group Id required. Please pass as query.properties.groupId"
+        );
+      }
+    });
+  });
+  describe("executes search:", () => {
+    let qry: IQuery;
+    beforeEach(() => {
+      qry = {
+        targetEntity: "groupMember",
+        filters: [
+          {
+            predicates: [
+              {
+                name: "steve",
+                remove: "property",
```

Comment:
Shouldn't we test the search spy result to make sure that this property is _actually_ removed? The bug I found earlier would have actually propagated this property to `serializeQueryForPortal`. I suspect the property would've been removed at that point, but it doesn't hurt to verify.